test/spec/ol/source/rastersource.test.js

```diff
@@ -0,0 +1,307 @@
+goog.provide('ol.test.source.RasterSource');
+
+var red = 'data:image/gif;base64,R0lGODlhAQABAPAAAP8AAP///yH5BAAAAAAALAAAAAA' +
+    'BAAEAAAICRAEAOw==';
+
+var green = 'data:image/gif;base64,R0lGODlhAQABAPAAAAD/AP///yH5BAAAAAAALAAAA' +
+    'AABAAEAAAICRAEAOw==';
+
+var blue = 'data:image/gif;base64,R0lGODlhAQABAPAAAAAA/////yH5BAAAAAAALAAAAA' +
+    'ABAAEAAAICRAEAOw==';
+
+function itNoPhantom() {
```

Comment:
Yes.  I'd like to see if we can provide a simple way to mark tests as not runnable in Phantom (this is not limited to just rendering tests).  In addition, I think it makes sense to run tests in real browsers (e.g. via Sauce Labs).  And we should be able to get rid of the build step required before running tests.  But I think it makes sense to do all this separately.